node/cmd/main.go

```diff
@@ -59,6 +61,14 @@ func NodeMain(ctx *cli.Context) error {
 		return err
 	}
 
+	if config.GCSafetyBufferSizeGB > 0 {
+		safetyBuffer := uint64(config.GCSafetyBufferSizeGB * float64(units.GiB))
```

Comment:
the field has `GB` suffix, so I don't think you should be converting to `GiB` again here (or the config field should be renamed)

Reply:
Nevermind, I'm dumb